server-src/scripts/setUpDB.sh

```diff
@@ -0,0 +1,36 @@
+#Setup db for kmt-prototype
+
+#Clear user details from db
+mongo kmt_db --eval 'db.user.remove({ })'
+
+#Clear role details from db
+mongo kmt_db --eval 'db.role.remove({ })'
+
+
+#Create admin account
+#Table = user
+mongo kmt_db --eval 'db.user.insert(
+     {
+         "email": "kmtAdmin@mailinator.com",
+         "userRole": "admin",
+         "firstName": "kmt",
+         "lastName": "admin",
+         "password": "Admin123",
+         "status": "A"
```

Comment:
status - we are thinking of having boolean value
true - active
false - inactive